server/issuers.go

```diff
@@ -53,7 +53,14 @@ type issuerFetchRequestV2 struct {
 
 // GetLatestIssuer - get the latest issuer by type/cohort
 func (c *Server) GetLatestIssuer(issuerType string, issuerCohort int16) (*model.Issuer, *handlers.AppError) {
-	issuer, err := c.fetchIssuersByCohort(issuerType, issuerCohort)
+	issuer, err := c.fetchIssuersByCohort(
```

Comment:
question: This might still not work as this method is [used](https://github.com/brave-intl/challenge-bypass-server/blob/master/server/issuers.go#L130-L152) in the get issuers v3 endpoint which is [used](https://github.com/brave-intl/bat-go/blob/master/services/skus/credentials.go#L157-L159) in SKUs when creating an issuer.